src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -859,6 +859,7 @@
       <Link>Resources\%(RecursiveDir)%(Filename)%(Extension)</Link>
       <LogicalName>Microsoft.Data.SqlClient.Resources.%(Filename).resources</LogicalName>
     </EmbeddedResource>
+    <Compile Remove="C:\Users\mdaigle\SqlClient\src\..\src\Microsoft.Data.SqlClient\src\Microsoft\Data\SqlClient\SSPI\ManagedSSPIContextProvider.cs" />
```

Comment:
Do you intend to delete the file here, or just to exclude it from the csproj?

Reply:
Not sure how that got in there. Didn't mean to include this.